renderer/components/preferences/categories/settings.js

```diff
@@ -146,6 +147,9 @@ class Settings extends React.Component {
         <Item title="Save to…" subtitle={kapturesDirPath} tooltip={kapturesDir} onSubtitleClick={this.openKapturesDir}>
           <Button title="Choose" onClick={pickKapturesDir}/>
         </Item>
+        <Item subtitle="Show cropper on launch">
```

Comment:
You need a title too. We only do subtitle-only items when they are subitems of the above item.

Reply:
Or just change it to title without subtitle.

Reply:
👍 It could probably use a subtitle explaining when it wouldn’t open despite the setting.